docs/quick_start.md

```diff
@@ -1,17 +1,157 @@
-| torch       | >= 2.4.0          | [2.5.1](https://github.com/pytorch/pytorch/releases/tag/v2.5.1)      | Required for torch-npu and vllm |
+### Prepare Environment
+
+You can use the container image directly with one line command:
+
+```bash
+DEVICE=/dev/davinci7
+IMAGE=quay.io/ascend/cann:8.0.rc3.beta1-910b-ubuntu22.04-py3.10
+docker run \
+    --name $NAME --device vllm-ascend-env \
+    --device /dev/davinci_manager --device /dev/devmm_svm --device /dev/hisi_hdc \
+    -v /usr/local/dcmi:/usr/local/dcmi -v /usr/local/bin/npu-smi:/usr/local/bin/npu-smi \
+    -v /usr/local/Ascend/driver/lib64/:/usr/local/Ascend/driver/lib64/ \
+    -v /usr/local/Ascend/driver/version.info:/usr/local/Ascend/driver/version.info \
+    -v /etc/ascend_install.info:/etc/ascend_install.info \
+    -it --rm $IMAGE bash
+```
+
+You can verify by running below commands in above container shell:
+
+```bash
+npu-smi info
+```
+
+You will see following message:
+
+```
++-------------------------------------------------------------------------------------------+
+| npu-smi 23.0.2              Version: 23.0.2                                               |
++----------------------+---------------+----------------------------------------------------+
+| NPU   Name           | Health        | Power(W)    Temp(C)           Hugepages-Usage(page)|
+| Chip                 | Bus-Id        | AICore(%)   Memory-Usage(MB)  HBM-Usage(MB)        |
++======================+===============+====================================================+
+| 0     xxx            | OK            | 0.0         40                0    / 0             |
+| 0                    | 0000:C1:00.0  | 0           882  / 15169      0    / 32768         |
++======================+===============+====================================================+
+```
+
+
+## 2. Installation
+
+Prepare:
+
+```bash
+apt update
+apt install git curl vim -y
+# Config pypi mirror to speedup
+pip config set global.index-url https://mirrors.tuna.tsinghua.edu.cn/pypi/web/simple
+```
+
+Create your venv
+
+```bash
+python3 -m venv .venv
+source .venv/bin/activate
+```
+
+You can install vLLM and vllm-ascend plugin by using:
+
+```bash
+# Install vLLM main branch (About 5 mins)
+git clone --depth 1 https://github.com/vllm-project/vllm.git
+cd vllm
+VLLM_TARGET_DEVICE=empty pip install .
+cd ..
+
+# Install vLLM Ascend Plugin:
+git clone --depth 1 https://github.com/vllm-project/vllm-ascend.git
+cd vllm-ascend
+pip install -e .
+cd ..
+```
+
+
+## 3. Usage
+
+After vLLM and vLLM Ascend plugin installation, you can start to
+try [vLLM QuickStart](https://docs.vllm.ai/en/latest/getting_started/quickstart.html).
+
+You have two ways to start vLLM on Ascend NPU:
+
+### Offline Batched Inference with vLLM
+
+With vLLM installed, you can start generating texts for list of input prompts (i.e. offline batch inferencing).
+
+```bash
+# Use Modelscope mirror to speed up download
+pip install modelscope
+export VLLM_USE_MODELSCOPE=true
+$ python3
```

Comment:
what is this used for?

Reply:
previous thoughts was a step by step cmd but I think it's OK to remove and add notes